src/components/PartnersContent/PartnersContent.test.tsx

```diff
@@ -0,0 +1,209 @@
+      websiteLink: "website2.com",
+    },
+  },
+  {
+    slug: "test-data-3",
+    attributes: {
+      companyName: "test data 3",
+      description: "Some Company description",
+      logo: "/logo3",
+      typeOfPartner: "Partners",
+      websiteLink: "website3.com",
+    },
+  },
+];
+
+const mockData2 = [
+  {
+    slug: "test-data-1",
+    attributes: {
+      companyName: "test data 1",
+      description: "Some Company description",
+      logo: "/logo1",
+      typeOfPartner: "Platform",
+      websiteLink: "website1.com",
+    },
+  },
+  {
+    slug: "test-data-2",
+    attributes: {
+      companyName: "test data 2",
+      description: "Some Company description",
+      logo: "/logo2",
+      typeOfPartner: "Platform",
+      websiteLink: "website2.com",
+    },
+  },
+];
+
+const mockData3 = [
+  {
+    slug: "test-data-3",
+    attributes: {
+      companyName: "test data 3",
+      description: "Some Company description",
+      logo: "/logo3",
+      typeOfPartner: "Partners",
+      websiteLink: "website3.com",
+    },
+  },
+  {
+    slug: "test-data-4",
+    attributes: {
+      companyName: "test data 4",
+      description: "Some Company description",
+      logo: "/logo4",
+      typeOfPartner: "Partners",
+      websiteLink: "website4.com",
+    },
+  },
+];
+
+const mockData4 = [
+  {
+    slug: "a",
+    attributes: {
+      companyName: "a",
+      description: "Some Company description",
+      logo: "/logo3",
+      typeOfPartner: "Partners",
+      websiteLink: "website3.com",
+    },
+  },
+  {
+    slug: "y",
+    attributes: {
+      companyName: "y",
+      description: "Some Company description",
+      logo: "/logo3",
+      typeOfPartner: "Platform",
+      websiteLink: "website3.com",
+    },
+  },
+  {
+    slug: "r",
+    attributes: {
+      companyName: "r",
+      description: "Some Company description",
+      logo: "/logo3",
+      typeOfPartner: "Partners",
+      websiteLink: "website3.com",
+    },
+  },
+  {
+    slug: "b",
+    attributes: {
+      companyName: "b",
+      description: "Some Company description",
+      logo: "/logo3",
+      typeOfPartner: "Platform",
+      websiteLink: "website3.com",
+    },
+  },
+  {
+    slug: "c",
+    attributes: {
+      companyName: "c",
+      description: "Some Company description",
+      logo: "/logo3",
+      typeOfPartner: "Partners",
+      websiteLink: "website3.com",
+    },
+  },
+];
+
+describe("PartnersContent", () => {
+  it("should render the page correctly, given the partners' data", () => {
+    render(<PartnersContent partnersData={mockData1} />);
+
+    expect(screen.getByText("Platform")).not.toBeUndefined;
+    expect(screen.getByText("Partners")).not.toBeUndefined;
+    expect(screen.getByTestId("partner-logo-test data 1")).not.toBeUndefined;
+    expect(screen.getByTestId("partner-logo-test data 2")).not.toBeUndefined;
+    expect(screen.queryByTestId("partner-logo-test data 3")).toBeUndefined;
```

Comment:
why are testid not hyphen case and being uniquely asserted here? what is tested here? it is passing regardless of `toBeUndefined` and `not.toBeUndefined` for all assertions 👀

Reply:
ok fixed